test/e2e/apps/daemon_set.go

```diff
@@ -108,7 +108,12 @@ var _ = SIGDescribe("Daemon set [Serial]", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
-	It("should run and stop simple daemon", func() {
+	/*
+	  Testname: DaemonSet-Creation
+	  Description: A conformant Kubernetes distribution MUST support the creation of DaemonSets. When a DaemonSet
+	  Pod is deleted, the DaemonSet controller MUST recreate the Pod.
```

Comment:
nit: we never "recreate" a Pod but simply create a new one.

Reply:
making the distinction makes sense to me

Reply:
I think this terminology is ok in this case because the pod is associated with the node. The terminology I generally discourage is "reschedule".

Reply:
I'll go with Janet's suggestion because its conformance and someone might infer that we are updating the Pod in place. In general I wouldn't read "recreate" as in place update, but it can't hurt.